modular_zubbers/code/game/machinery/crew_monitor.dm

```diff
@@ -16,14 +16,16 @@
 
 	for(var/tracked_mob in GLOB.suit_sensors_list)
 		var/mob/living/carbon/human/mob = tracked_mob
+		if(!istype(mob))
+			continue
 		if(mob.z != src.z  && !HAS_TRAIT(mob, TRAIT_MULTIZ_SUIT_SENSORS))
 			continue
 		var/obj/item/clothing/under/uniform = mob.w_uniform
-		if(uniform.sensor_mode >= SENSOR_VITALS && HAS_TRAIT(mob, TRAIT_CRITICAL_CONDITION) || mob.stat == DEAD && mob.mind)
-			canalarm = TRUE
-		else if(uniform.sensor_mode == SENSOR_LIVING && mob.stat == DEAD)
```

Comment:
The idea was that your lazy paramedic was supposed to use it as a cue to get their rear moving, ideally roaming the station looking for said person, instead of going "meh no locaction their fault"

Reply:
The issue is that these sensors cover not only medbay, but also: Security, Bridge, sometimes even checkpoints. Beside, as a paramed, you use the handheld version